packages/cli/src/commands/network/parameters.ts

```diff
@@ -0,0 +1,106 @@
+import { BaseCommand } from '../../base'
+
+export default class Parameters extends BaseCommand {
+  static description = 'View network parameters'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  async run() {
+    const goldToken = await this.kit.contracts.getGoldToken()
+    const stableToken = await this.kit.contracts.getStableToken()
+    const attestations = await this.kit.contracts.getAttestations()
+    console.log(
```

Comment:
please, investigate using some of the `cli` output function, instead of console log. 

For example, we do:
```ts
    const balances = {
      goldBalance: await goldToken.balanceOf(args.account),
      dollarBalance: await stableToken.balanceOf(args.account),
    }
    printValueMap(balances)
```